db/controllers/controller.js

```diff
@@ -0,0 +1,38 @@
+const { request } = require("../app.js");
+const {
+  selectTopics,
+  selectArticleById,
+  selectArticles,
+} = require("../models/model.js");
+
+function getTopics(req, res, next) {
+  return selectTopics().then((topics) => {
+    res.status(200).send({ topics });
+  });
+}
+
+function getArticles(req, res, next) {
+  //   let sort_by, order;
+  //   if (req.query) {
+  //     sort_by = req.query.sort_by;
+  //     order = req.query.order;
+  //   }
+  return selectArticles() //sort_by, order
+    .then((articles) => {
+      res.status(200).send({ articles });
+    })
+    .catch((err) => console.log(err));
```

Comment:
console logging is not actually handling your error, in order to pass this error into the error-handling-middleware chain you will need to invoke `next(err)`